exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,31 @@
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
```

Comment:
It turns out you can not, due to lexical scope issues. Sorry for misleading you.

http://blog.honeybadger.io/understanding-ruby-refinements-and-lexical-scope/